### ISSUE TITLE
[win11] Closing the launcher minimises to system tray then allows starting more than one instance

### ISSUE THREAD

**Issue by ThetaSinner** [holochain/launcher-tauri#153]
**Describe the bug**

When I close the launcher on Windows it actually minimises to the system tray. There is no indication that this is the intended behaviour of the close button and thinking its closed, when I then try to launch a new instance it will never load.

**To Reproduce**

Start the launcher
![first-launch](https://user-images.githubusercontent.com/6267702/229464604-ccd62c06-fd57-42d2-9d5f-469675a1dfb5.png)

Then close it using the close button on the main window
![close-app](https://user-images.githubusercontent.com/6267702/229464789-b5c39acf-1c77-495f-9668-568fcbfbbf91.png)

This leaves everything running, but not visible (it's inside the 'Show hidden icons' section of my system tray)
![stays-running-in-system-tray](https://user-images.githubusercontent.com/6267702/229464836-ac293c2c-18bc-4563-8647-1d97a4fe4adf.png)

Now use the start menu or desktop icon to launch a second instance of the launcher. This results in two instances
![launch-from-start-menu-two-copies](https://user-images.githubusercontent.com/6267702/229465161-3d27c6bb-dba2-4c8c-bbab-190cf6b2345f.png)

If you try to log into the second instance it will never load, it just stays on the loading screen
![one-stays-loading](https://user-images.githubusercontent.com/6267702/229465346-a876db1f-fc8e-4784-84d1-a729c67eb17c.png)

To get back to a good state, I have to either quit from the system tray (if I know it's there) or kill from the task manager to get back to a state where nothing is running
![kill-from-task-manager](https://user-images.githubusercontent.com/6267702/229465671-3d2bed7a-9525-43d5-8927-05208b3720b4.png)

Then I can get back to a good state where only one instance is running
![back-to-okay-after-kill](https://user-images.githubusercontent.com/6267702/229465729-61073d1f-1bda-432b-829a-6ba4a1402d72.png)

**Expected behavior**
- When I try to launch a second instance, the already running launcher should be restored from the system tray rather than starting a new instance.
- It would be nice to be notified the first time I try to close the window that it is going to the system tray and be given an explanation of why it wants to keep running.

**Logs**

The launchers share a log file, but this appears to be all that comes out of the second instance
```text
[2023-04-03T10:16:24.345396800+01:00] INFO - Launch setup successful
[2023-04-03T10:16:27.031010+01:00] ERROR - [LAIR] {"error":"EIO"}
[2023-04-03T10:16:27.033965600+01:00] INFO - [LAIR] Terminated(TerminatedPayload { code: Some(0), signal: None })
```

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Holochain Launcher Version: v0.9.4 with holochain beta v0.1.3 and lair keystore v0.2.3

**Comment by matthme** [comment]
Yes this is indeed especially confusing on Windows since the system tray is not really visible. A note asking whether to keep the Launcher running in the background or quit it completely would probably be better.
It is however intended that multiple Launcher instances can run in parallel, potentially with different profiles. Maybe a dialog asking whether to open the running instance or a new one would be ideal. And obviously it should not fail on startup when trying to launch a second instance.

**Comment by matthme** [comment]
The part where opening the launcher if there is already an instance running fails has been addressed by using `tauri-plugin-single-instance` (69372b2) and will be part of the next release.

**Comment by ThetaSinner** [comment]
Awesome! Looking forward to having that :)
